babel.config.js

```diff
@@ -5,11 +5,18 @@
 
 // babelrc doesn't respect NODE_PATH anymore but using require does.
 // Alternative to install them locally in node_modules
-module.exports = {
-  presets: [require("@babel/preset-env"), require("@babel/preset-react"), require("@babel/preset-typescript")],
-  plugins: [
-    [require("@babel/plugin-transform-runtime"), { regenerator: true }],
-    require("@babel/plugin-proposal-class-properties"),
-    require("@babel/plugin-proposal-object-rest-spread"),
-  ],
+module.exports = function (api) {
+  // ensure env is test so that this config won't impact build or dev server
+  if (api.env("test")) {
```

Comment:
I would love to figure a way out that you don't even need this in your test env.

Reply:
that would be great if we can remove this file totally.